### ISSUE TITLE
API docs stopped working on nightly (and local) build

### ISSUE THREAD

**Issue by jodator** [ckeditor/ckeditor5#6613]
## 📝 Provide a description of requested docs changes

The API list item cannot be expandend. This error happens:

![](https://user-images.githubusercontent.com/247363/79422811-d0fcdb00-7fbd-11ea-81ef-c48274cde8ac.png)

Details:

```
jquery.js:4046 jQuery.Deferred exception: n.hasOwnProperty is not a function TypeError: n.hasOwnProperty is not a function
    at HTMLLIElement.<anonymous> (https://ckeditor5.github.io/docs/nightly/assets/1.2.3/scripts/app.js:46:700)
    at Function.each (https://ckeditor5.github.io/docs/nightly/assets/1.2.3/scripts/app.js:14:2952)
    at T.fn.init.each (https://ckeditor5.github.io/docs/nightly/assets/1.2.3/scripts/app.js:14:1428)
    at t (https://ckeditor5.github.io/docs/nightly/assets/1.2.3/scripts/app.js:46:640)
    at https://ckeditor5.github.io/docs/nightly/assets/1.2.3/scripts/app.js:46:1668
    at l (https://ckeditor5.github.io/docs/nightly/assets/1.2.3/scripts/app.js:46:1672)
    at HTMLDocument.<anonymous> (https://ckeditor5.github.io/docs/nightly/assets/1.2.3/scripts/app.js:95:7)
    at u (https://ckeditor5.github.io/docs/nightly/assets/1.2.3/scripts/app.js:25:26157)
    at f (https://ckeditor5.github.io/docs/nightly/assets/1.2.3/scripts/app.js:25:26456) undefined
T.Deferred.exceptionHook @ jquery.js:530
jquery.js:4055 Uncaught TypeError: n.hasOwnProperty is not a function
    at HTMLLIElement.<anonymous> (_filtering.js:51)
    at Function.each (jquery.js:381)
    at T.fn.init.each (jquery.js:203)
    at t (_filtering.js:47)
    at _filtering.js:101
    at l (_filtering.js:21)
    at HTMLDocument.<anonymous> (app.js:28)
    at u (jquery.js:3762)
    at f (jquery.js:3830)
```

**Comment by jodator** [comment]
Might be related to recent jQuery version bump from `3.4.1` to `3.5.0`.

**Comment by pomek** [comment]
Fixed with the latest Umberto release and ba5965179e8632e0128a182aaf7289d3f26257de.
